corehq/apps/hqwebapp/views.py

```diff
@@ -153,6 +154,9 @@ def server_error(request, template_name='500.html'):
     traceback_key = uuid.uuid4().hex
     cache.cache.set(traceback_key, traceback_text, 60*60)
 
+    if settings.UNIT_TESTING:
+        return HttpResponse(status=500)
```

Comment:
(copying from commit message) 

a rendering error on the error page itself just creates a huge stack trace because it recursively calls itself making it hard to debug in tests. I think this makes more sense and will save someone time in the future, though I don't love `UNIT_TESTING` references in code that gets run in production so open to push back...

Reply:
Agreed about not loving `UNIT_TESTING` references in code. Is this just an issue when you make a mistake when changing the error page? I think if you decide to keep this, it a short comment about why its here may be helpful

Reply:
It's specifically related to this `'Settings' object has no attribute 'STATICI18N_OUTPUT_DIR'` message which triggers on rendering of any template that includes the `statici18n` templatetag (which is basically all our base templates). So any view that fails with this error tries to render the 500 page which then also fails with this error, and then you get a stack explosion.

Reply:
Added comment in 91d2055895e6b7eacdec1604e5d2482ef60c0e42